assets/components/flashSaleCountdown/flashSaleCountdown.scss

```diff
@@ -0,0 +1,20 @@
+.component-flash-sale-countdown__chip {
+  font-family: $gu-text-sans-web;
+  font-size: 18px;
+  padding: 6px 10px 4px;
+  display: inline-block;
+}
+
+.component-flash-sale-countdown__chip--time {
+  background: gu-colour(sport-garnett-feature);
+  color: #fff;
+  font-weight: 800;
+  width: 98px;
```

Comment:
`98px` feels very magical. Could we add a comment for it?

Reply:
Haha, i just dropped an inline comment here, let me put it in the CSS itself